test/framework/nutanix.go

```diff
@@ -17,28 +17,40 @@ import (
-	nutanixPrismElementClusterName  = "T_NUTANIX_PRISM_ELEMENT_CLUSTER_NAME"
-	nutanixSSHAuthorizedKey         = "T_NUTANIX_SSH_AUTHORIZED_KEY"
-	nutanixSubnetName               = "T_NUTANIX_SUBNET_NAME"
-	nutanixControlPlaneEndpointIP   = "T_NUTANIX_CONTROL_PLANE_ENDPOINT_IP"
-	nutanixControlPlaneCidrVar      = "T_NUTANIX_CONTROL_PLANE_CIDR"
-	nutanixPodCidrVar               = "T_NUTANIX_POD_CIDR"
-	nutanixServiceCidrVar           = "T_NUTANIX_SERVICE_CIDR"
-	nutanixTemplateNameUbuntu123Var = "T_NUTANIX_TEMPLATE_NAME_UBUNTU_1_23"
-	nutanixTemplateNameUbuntu124Var = "T_NUTANIX_TEMPLATE_NAME_UBUNTU_1_24"
-	nutanixTemplateNameUbuntu125Var = "T_NUTANIX_TEMPLATE_NAME_UBUNTU_1_25"
-	nutanixTemplateNameUbuntu126Var = "T_NUTANIX_TEMPLATE_NAME_UBUNTU_1_26"
-	nutanixTemplateNameUbuntu127Var = "T_NUTANIX_TEMPLATE_NAME_UBUNTU_1_27"
-	nutanixTemplateNameUbuntu128Var = "T_NUTANIX_TEMPLATE_NAME_UBUNTU_1_28"
+	nutanixEndpoint                  = "T_NUTANIX_ENDPOINT"
+	nutanixPort                      = "T_NUTANIX_PORT"
+	nutanixAdditionalTrustBundle     = "T_NUTANIX_ADDITIONAL_TRUST_BUNDLE"
+	nutanixInsecure                  = "T_NUTANIX_INSECURE"
+	nutanixMachineBootType           = "T_NUTANIX_MACHINE_BOOT_TYPE"
+	nutanixMachineMemorySize         = "T_NUTANIX_MACHINE_MEMORY_SIZE"
+	nutanixSystemDiskSize            = "T_NUTANIX_SYSTEMDISK_SIZE"
+	nutanixMachineVCPUsPerSocket     = "T_NUTANIX_MACHINE_VCPU_PER_SOCKET"
+	nutanixMachineVCPUSocket         = "T_NUTANIX_MACHINE_VCPU_SOCKET"
+	nutanixPrismElementClusterName   = "T_NUTANIX_PRISM_ELEMENT_CLUSTER_NAME"
+	nutanixSSHAuthorizedKey          = "T_NUTANIX_SSH_AUTHORIZED_KEY"
+	nutanixSubnetName                = "T_NUTANIX_SUBNET_NAME"
+	nutanixControlPlaneEndpointIP    = "T_NUTANIX_CONTROL_PLANE_ENDPOINT_IP"
+	nutanixControlPlaneCidrVar       = "T_NUTANIX_CONTROL_PLANE_CIDR"
+	nutanixPodCidrVar                = "T_NUTANIX_POD_CIDR"
+	nutanixServiceCidrVar            = "T_NUTANIX_SERVICE_CIDR"
+	nutanixTemplateNameUbuntu123Var  = "T_NUTANIX_TEMPLATE_NAME_UBUNTU_1_23"
+	nutanixTemplateNameUbuntu124Var  = "T_NUTANIX_TEMPLATE_NAME_UBUNTU_1_24"
+	nutanixTemplateNameUbuntu125Var  = "T_NUTANIX_TEMPLATE_NAME_UBUNTU_1_25"
+	nutanixTemplateNameUbuntu126Var  = "T_NUTANIX_TEMPLATE_NAME_UBUNTU_1_26"
+	nutanixTemplateNameUbuntu127Var  = "T_NUTANIX_TEMPLATE_NAME_UBUNTU_1_27"
+	nutanixTemplateNameUbuntu128Var  = "T_NUTANIX_TEMPLATE_NAME_UBUNTU_1_28"
+	nutanixTemplateNameRedHat123Var  = "T_NUTANIX_TEMPLATE_NAME_REDHAT_1_23"
+	nutanixTemplateNameRedHat124Var  = "T_NUTANIX_TEMPLATE_NAME_REDHAT_1_24"
+	nutanixTemplateNameRedHat125Var  = "T_NUTANIX_TEMPLATE_NAME_REDHAT_1_25"
+	nutanixTemplateNameRedHat126Var  = "T_NUTANIX_TEMPLATE_NAME_REDHAT_1_26"
+	nutanixTemplateNameRedHat127Var  = "T_NUTANIX_TEMPLATE_NAME_REDHAT_1_27"
+	nutanixTemplateNameRedHat128Var  = "T_NUTANIX_TEMPLATE_NAME_REDHAT_1_28"
+	nutanixTemplateNameRedHat9123Var = "T_NUTANIX_TEMPLATE_NAME_REDHAT_9_1_23"
```

Comment:
This isn't needed anymore either for the same reason

Reply:
Fixed